src/components/Filters/FiltersHydrator.js

```diff
@@ -1,66 +1,35 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment, useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import noop from 'lodash/noop';
+import { useSelector, useDispatch } from 'react-redux';
+import { selectors, actions } from '../../redux/modules/filters';
 
-import { actions, selectors } from '../../redux/modules/filters';
+const FiltersHydrator = ({ namespace, onHydrated }) => {
+    
+  const isHydrated  = useSelector(state => selectors.getIsHydrated(state.filters, namespace));
+  const [hydrated, setHydrated] = useState(false);
 
-class FiltersHydrator extends Component {
-  static propTypes = {
-    hydrateFilters: PropTypes.func.isRequired,
-    filtersHydrated: PropTypes.func.isRequired,
-    namespace: PropTypes.string.isRequired,
-    onHydrated: PropTypes.func,
-    isHydrated: PropTypes.bool
-  };
+  const dispatch = useDispatch();
 
-  static defaultProps = {
-    onHydrated: noop,
```

Comment:
where are defaults?

Reply:
no need, just checking inside that the method exists before usage